### PR TITLE
Control grid option via comment

### PR DESCRIPTION
**Pull request by fanich37**
* Add gridStatus method to Processor

* Add test case

* Update README.md

### CODE REVIEW

**Comment by ai** [comment]
@fanich37 awesome! One last thing. You need to improve test. Right now tests doesn’t execute every line of code.

Run:

```
npx jest --coverage
```

It will show what lines was not executed

**Comment by ai** [comment]
Seems like yoou need to test `result.warn` call on double `autoprefixer grid` comments.

**Comment by Dan503** [comment]
Can you also add an `/* autoprefixer grid: off */` command?

It should work both ways, you can turn the setting on if you have grid disabled by default or off if you have it enabled by default.

BTW, I ❤ the idea 😊

**Comment by Dan503** [comment]
Oh, you did, you just didn't add it to the readme.

**Comment by fanich37** [comment]
> Seems like yoou need to test `result.warn` call on double `autoprefixer grid` comments.

  @ai Sure, I'll add them 👌

But, to be honest, I don't like the behaviour of `gridStatus` method. What I mean - during the processor execution `gridStatus` fires several times and it's ok if the rule itself has two or more control comments. But if these control comments are located in the root of the doc - then the console gets polluted with identical warnings. I'll try to figure out the solution for it.

**Comment by fanich37** [comment]
> Oh, you did, you just didn't add it to the readme.
> 
> Can you add the off command to the readme?

@Dan503 Sure, I'll add it in the next commit 👌

File: lib/processor.js [hunk 1]
```diff
@@ -540,6 +540,64 @@ class Processor {
...
+  gridStatus (node, result) {
+    if (!node) return false
+
+    if (node._autoprefixerGridStatus !== undefined) {
+      return node._autoprefixerGridStatus
+    }
+
+    if (node.parent) {
+      let p = node.prev()
+      if (p && p.type === 'comment' && IGNORE_NEXT.test(p.text)) {
```
Comment (ai) [inline lib/processor.js]: А нам нужно это? Мы же всегда вызываем `disabled()` до запуска `gridStatus`.

File: lib/processor.js [hunk 2]
```diff
@@ -540,6 +540,64 @@ class Processor {
...
+        return false
+      }
+    }
+
+    let value = null
+    if (node.nodes) {
```
Comment (ai) [inline lib/processor.js]: `node` — это свойство, к которому мы ходим добавить префикс? Зачем тогда мы смотри его детей? Надо смотреть `node.parent.nodes`

**Comment by ai** [comment]
> But, to be honest, I don't like the behaviour of gridStatus method. What I mean - during the processor execution gridStatus fires several times

It creates `_autoprefixerGridStatus` cache, so multiple calls will be fast

**Comment by fanich37** [comment]
> > But, to be honest, I don't like the behaviour of gridStatus method. What I mean - during the processor execution gridStatus fires several times
> 
> It creates `_autoprefixerGridStatus` cache, so multiple calls will be fast

Yes, I added it in my last commit right after I realised what it does 😃

**Comment by Dan503** [comment]
@fanich37 I am writing an article about all the awesome new stuff that is in version 9.2.0.

Are you happy for me to call you by the name "Andrey Alexandrov" in the article?

Where would you like your name to link to? At the moment I'm linking to [your GitHub account](https://github.com/fanich37). Is there somewhere else you would prefer I link to like a personal website, your linked in profile, or your twitter profile?

**Comment by fanich37** [comment]
> @fanich37 I am writing an article about all the awesome new stuff that is in version 9.2.0.
> 
> Are you happy for me to call you by the name "Andrey Alexandrov" in the article?
> 
> Where would you like your name to link to? At the moment I'm linking to [your GitHub account](https://github.com/fanich37). Is there somewhere else you would prefer I link to like a personal website, your linked in profile, or your twitter profile?

Wow! That's awesome! Of course you can call me by name. And my GitHub account is the only social thing I'm actively using😃 Thank you, Daniel🙏

**Pull request merged**